integrated_channels/utils.py

```diff
@@ -247,7 +247,6 @@ def generate_formatted_log(
     Arguments:
     - channel_name (str): The name of the integrated channel
     - enterprise_customer_uuid (str): UUID of the relevant EnterpriseCustomer
-    - enterprise_customer_identifer (str): Identifying name of the EnterpriseCustomer
```

Comment:
Thanks for the catch here!

Reply:
nw, I figured I would take care of this while I was here and the editor was pointing it out too, so great.